addons/website/views/website_templates.xml

```diff
@@ -259,1105 +259,1166 @@
+    <button type="button" t-attf-class="navbar-toggler #{_toggler_class} o_not_editable" data-bs-toggle="offcanvas" data-bs-target="#top_menu_collapse">
         <span class="navbar-toggler-icon o_not_editable"/>
     </button>
 </template>
 
 <template id="navbar" name="Navbar">
     <t t-set="_navbar_expand_class" t-value="_navbar_expand_class is None and 'navbar-expand-lg' or ''"/>
-    <nav data-name="Navbar" t-attf-class="navbar #{_navbar_expand_class} navbar-light o_colored_level o_cc #{_navbar_classes}">
+    <nav data-name="Navbar" t-attf-class="navbar #{_navbar_expand_class} navbar-light o_colored_level o_cc #{_navbar_classes} #{_extra_navbar_classes}">
         <t t-out="0"/>
     </nav>
 </template>
 
 <template id="navbar_nav" name="Navbar Nav">
-    <ul id="top_menu" t-attf-class="nav navbar-nav o_menu_loading #{_nav_class}">
+    <ul id="top_menu" t-attf-class="nav navbar-nav #{'' if _no_autohide_menu_mobile else 'o_menu_loading'} #{_nav_class}">
         <t t-out="0"/>
     </ul>
 </template>
 
 <!-- Header templates -->
 
-<!-- "Default" template -->
-<template id="template_header_default" inherit_id="website.layout" name="Template Header Default" active="True">
-    <xpath expr="//header//nav" position="replace">
-        <t t-call="website.navbar">
-            <t t-set="_navbar_classes" t-valuef="shadow-sm"/>
-            <div id="top_menu_container" class="container justify-content-start justify-content-lg-between">
-                <!-- Brand -->
-                <t t-call="website.placeholder_header_brand">
-                    <t t-set="_link_class" t-valuef="me-4"/>
-                </t>
-                <!-- Navbar Collapse -->
-                <div id="top_menu_collapse" class="collapse navbar-collapse order-last order-lg-0">
-                    <t t-call="website.navbar_nav">
-                        <t t-set="_nav_class" t-valuef="flex-grow-1"/>
-                        <!-- Menu -->
-                        <t t-foreach="website.menu_id.child_id" t-as="submenu">
-                            <t t-call="website.submenu">
-                                <t t-set="item_class" t-valuef="nav-item"/>
-                                <t t-set="link_class" t-valuef="nav-link"/>
+<!-- "mobile" template -->
+<template id="template_header_mobile" name="Template Header Mobile">
+    <t t-call="website.navbar">
+        <t t-set="_navbar_classes" t-valuef="o_header_mobile d-block d-lg-none shadow-sm"/>
+        <t t-set="_navbar_expand_class" t-valuef="None"/>
+        <div id="top_menu_container" class="container flex-wrap justify-content-between">
+            <!-- Brand -->
+            <t t-call="website.placeholder_header_brand"/>
+            <ul class="list-unstyled d-flex align-items-center gap-2 mb-0">
+                <!-- Cart / Wishlist placeholder -->
+                <div id="o_cart_whish_placeholder_mobile"/>
```

Comment:
edit: In any case better to don't reference the wishlist module in the placeholder name, let's keep it more generic ;)

Reply:
> An `<ul>` can't contain a `<div>`.

@stefanorigano 
Since the Extra elements, Cart and Dropdown elements have different parent tags (`div` or `li`), and we sometimes need `div` instead of `li`, can I generalize all these elements with a `div` wrapper instead of `ul`/`li` ?

Reply:
Sure,  a `<nav>` tag would be even more appropriate IMO.

Reply:
Since the main wrap  tag is already a `<nav>` : 
```xml
<header>
<nav data-name="Navbar" class="navbar navbar-expand-lg ...">
...
</nav>
</header>

```
 I'm not sure it's a good idea to nest a '<nav>' inside another. What do you think? 🤔 
I think we should go with `div`

Reply:
As the Portal default template uses `ul/li`, I've changed the header templates to match it and used `nav` when it was possible.